frontend/src/api/open-hands.types.ts

```diff
@@ -70,12 +70,26 @@ export interface AuthenticateResponse {
   error?: string;
 }
 
+export enum ProviderType {
+  GITHUB = "github",
+  GITLAB = "gitlab",
+}
```

Comment:
We already have enums for this in `frontend/src/types/settings.ts` that can be reused here

Reply:
Thanks for your input!

Added a change to reuse the existing definition: https://github.com/All-Hands-AI/OpenHands/pull/7923/commits/34542d6e25b78a5996094e5101f763d418d11914